msal/src/Microsoft.Identity.Client/UIParent.cs

```diff
@@ -138,6 +128,37 @@ public static bool IsSystemWebviewAvailable()
                 return false;
             }
         }
+
+        private static bool IsBrowserWithCustomTabSupportAvailable()
```

Comment:
>IsBrowserWithCustomTabSupportAvailable [](start = 28, length = 38)

Did you validate this for all the cases you mention in your PR?

Reply:
not all...i did: chrome disabled, chrome enabled, chrome disabled w/firefox, chrome disabled w/firefox and edge

---
In reply to: [227939830](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1347#discussion_r227939830) [](ancestors = 227939830)

Reply:
ok. 

---
In reply to: [227941563](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1347#discussion_r227941563) [](ancestors = 227941563,227939830)